evalml/data_checks/id_columns_data_check.py

```diff
@@ -89,66 +89,81 @@ def validate(self, X, y=None):
             ...     }
             ... ]
 
-            If the first column of the dataframe is identified as an ID column it is most likely the primary key.
+            Despite being all unique, "Country_Rank" will not be identified as an ID column as id_threshold is set to 1.0
```

Comment:
thanks for clearing this up, this text is much more helpful.

NAB but is this confusing? If the threshold is set to 1, I would sort of expect it to be identified as ID whether or not that was including in the column header. Might be worth discussing later.

Reply:
Should we reverse the threshold measurement such that the lower it is, the more selective `ID` column identification is?